crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations.rs

```diff
@@ -372,8 +384,16 @@ impl ProtocolCryptographicData {
                 public_input,
                 data,
                 advance_request,
+                first_round_output,
                 ..
             } => {
+                let session_id = match bcs::from_bytes(&first_round_output)? {
+                    VersionedDwalletDKGFirstRoundPublicOutput::V1(output) => {
+                        let (_, session_id) =
+                            bcs::from_bytes::<(Vec<u8>, CommitmentSizedNumber)>(&output)?;
+                        session_id
+                    }
+                };
```

Comment:
This should only be in the v1 patchy case, add a todo here to check protocol config before